Lib/pathlib.py

```diff
@@ -45,240 +53,11 @@ def _is_wildcard_pattern(pat):
     # be looked up directly as a file.
     return "*" in pat or "?" in pat or "[" in pat
 
-
-class _Flavour(object):
-    """A flavour implements a particular (platform-specific) set of path
-    semantics."""
-
```

Comment:
Is there a way to minimize the diff and preserve line history for this code?  For example making a base class here and inheriting from it in PurePath.

I know history is not the foremost concern, but there is often value in finding who added what line for what reason, and git tooling is still not very smart for this.

Reply:
@merwok, I agree that we should clearly aim for the smallest diff possible in every PR in order to avoid code churn and preserve history. But surely concerns about history should not be guiding substantive decisions about code structure, such as e.g. influencing decisions around a class's mro?

Reply:
I guess that ship has sailed with the removal of accessors and much code being moved!

Reply:
I'm a big `git blame` fan and I try to avoid moving code around if I can help it! But I think adding another class to the MRO for the sake of a tidy history might be a bridge too far in this case.

One thing I could do to help reduce the diff is to rename `_pathmod` to `_flavour`?